airflow/bin/cli.py

```diff
@@ -325,6 +325,21 @@ def task_state(args):
     print(ti.current_state())
 
 
+def dag_state(args):
+    """
+    Returns the state of a DagRun at the command line.
+
+    >>> airflow dag_state tutorial 2015-01-01
+    running
+    """
+    dag = get_dag(args)
+    dr = DagRun.find(dag.dag_id, execution_date=args.execution_date)
+    if len(dr) > 0:
```

Comment:
There is a UNIQUE constraint on (dag_id, execution_date), so I don't think that we can ever get more than 1 entries. I'll change code to be more pythonic.